src/app/public/config.ts

```diff
@@ -64,7 +64,27 @@ export interface SkyuxConfigApp {
   title?: string;
 }
 
+export interface SkyuxConfigHostBBCheckout {
+  version: '2';
```

Comment:
@Blackbaud-TerryHelems is this is the correct value?

Reply:
Yes, just going with `2`.

Reply:
Unless we wanted to do `2.0`?

Reply:
`2` is fine. It's not locking us into anything.